quantum/observable/transforms/JordanWigner/JW.cpp

```diff
@@ -14,27 +14,12 @@
 #include <memory>
 #include "FermionOperator.hpp"
 #include "PauliOperator.hpp"
-#include "xacc_observable.hpp"
 
 namespace xacc {
 namespace quantum {
-template <typename T>
-bool ptr_is_a(std::shared_ptr<Observable> ptr) {
-  return std::dynamic_pointer_cast<T>(ptr) != nullptr;
-}
+
 std::shared_ptr<Observable>
-JW::transform(std::shared_ptr<Observable> Hptr_input) {
-
```

Comment:
I _think_ I did this because the algorithms in which we use the transforms have some pre-processing of sorts. More generally, I decided to put it back in case of a new application that doesn't rely on the already implemented algorithms.